internal/config/config.go

```diff
@@ -86,3 +94,151 @@ func CI() (string, string, string) {
 
 	return ciHost, ciUser, ciToken
 }
+
+// ConvertHCLtoTOML() converts an .hcl file to a .toml file
+func ConvertHCLtoTOML(path string, file string) {
+	oldconfig := path + "/" + file + ".hcl"
+	newconfig := path + "/" + file + ".toml"
+
+	_, err := os.Stat(oldconfig)
+	if os.IsNotExist(err) {
+		return
+	}
+
+	_, err = os.Stat(newconfig)
+	if err == nil {
+		return
+	}
+
+	// read in the old config HCL file and write out the new TOML file
+	viper.Reset()
+	viper.SetConfigName("lab")
+	viper.SetConfigType("hcl")
+	viper.AddConfigPath(path)
+	viper.ReadInConfig()
+	viper.SetConfigType("toml")
+	viper.WriteConfigAs(newconfig)
+
+	// delete the old config HCL file
+	err = os.Remove(oldconfig)
+	if err != nil {
+		fmt.Println("Warning: Could not delete old config file", oldconfig)
+	}
+
+	// HACK
```

Comment:
fwiw, it doesn't surprise me at all that we had to do this. Should be fine since lab configs have remained simple, essentially for this reason too.

Reply:
> I don't know why codecov is commenting every 5 lines on the stuff that isn't covered by tests ... Super noisy, sorry folks I try to find out how to disable that (unless it's just me?)

Hmm ... I'm by no means an expert in codecov.  We do have a few engineers who are familiar with it.  Let me see if I can ping someone to take a look.